packages/2018-housing-affordability/src/components/ExploreHousingPolicyImplementation/index.js

```diff
@@ -1,43 +1,179 @@
+        { selectedPolicy && <div className={policyContainer} />}
+        <div className={flexContainer}>
+          {tableData &&
+            tableData.length &&
+            insert(
+              selectedPolicy
+                ? tableData.findIndex(
+                    item => item.policy === selectedPolicy
+                  ) + 1
+                : 0,
+              selectedPolicy ? (
+                <SelectedPolicy
+                  data={selectedPolicyData}
+                  key={'selectedPolicyInset'}
+                />
+              ) : (
+                <div className={policyContainer} />
+              ),
+              tableData.map(
+                item =>
+                  item.policy === selectedPolicy ? (
+                    <PolicyText
+                      data={item}
+                      onClick={() => this.handleClick(item.policy)}
+                      selected
+                      key={item.policy}
+                    />
+                  ) : (
+                    <PolicyText
+                      data={item}
+                      onClick={() => this.handleClick(item.policy)}
+                      key={item.policy}
+                    />
+                  )
+              )
+            )}
+        </div>
 
-        <p>There are a number of different housing programs and policies in place throughout the Portland metro area.
-          They’re administered by various jurisdictions ranging from the city to the state level and at times,
+        <p>
+          There are a number of different housing programs and policies in place
+          throughout the Portland metro area. They’re administered by various
+          jurisdictions ranging from the city to the state level and at times,
           these policies can be tough to interpret.
-          <br />
-          <br />
-          In the spring and summer of 2018,<a href="http://www.hackoregon.org/">Hack Oregon</a> volunteers surveyed local governments to understand what
-          policies were being implemented. Taking inspiration from the <a href="https://www.housingconsortium.org/maps/">Housing Consortium in Seattle, WA,</a>
-          the Hack Oregon project team created a framework of 22 policies impacting housing affordability in the region.
-          To get the most accurate representation of policies being implemented, government websites were reviewed and
-          interviews with government employees were conducted in order to verify our findings.
-          <br />
-          <br />
-          <a href="https://docs.google.com/spreadsheets/d/1SVs9AQkw5guxEIuV1eGASKCg5miWhPn3UoZrALQpXLI/edit?usp=sharing">Explore Findings</a></p>
-
+        </p>
+        <p>
+          In the spring and summer of 2018,{' '}
+          <a href="http://www.hackoregon.org/">Hack Oregon</a> volunteers
+          surveyed local governments to understand what policies were being
+          implemented. Taking inspiration from the{' '}
```

Comment:
What a cool jsx quirk.

Reply:
Been using prettify and didn't actually realize what was going on at first - I guess it's just so that you can see the whitespace when you're reading the code.

Reply:
It's more nefarious than that. [JSX strips all newlines between elements](https://github.com/facebook/react/issues/1643). This is contrary to HTML which treats newlines as whitespace characters. So Prettier must be protecting you from losing the expected space between `the` and the following `a` element.